addons/main/cfgVehicles.hpp

```diff
@@ -106,12 +114,40 @@ class cfgVehicles {
         isGlobal = 0; //run on server
         isTriggerActivated = 1; //Wait for triggers
         //icon = QPATHTOF(UI\Icon_Module_Make_Unit_Handcuffed_ca.paa);
-        class Arguments {
-            class anomalyId {
+        // class Arguments {
+        //     class anomalyId {
+        //         displayName = "ID";
+        //         description = "$STR_anomaly_teleport_id_desc";
+        //         typeName = "NUMBER";
+        //         defaultValue = 0;
+        //     };
+        // };
+        class Attributes: AttributesBase {
+            class anomalyId: Edit {
                 displayName = "ID";
-                description = "$STR_anomaly_teleport_id_desc";
-                typeName = "NUMBER";
-                defaultValue = 0;
+                tooltip = "$STR_anomaly_teleport_id_desc";
+                property = "anomaly_teleport_id";
```

Comment:
```suggestion
                property = QGVAR(teleport_id);
```

Tho why need this attribute in the first place?

Reply:
This is used when you select multiple objects and edit their attributes all at once. it is used detect that it is the same attribute. I do not use CBA typically, so I had not thought to use QGVAR, but it does make sense to.